tests/tables-test.sps

```diff
@@ -630,11 +630,11 @@
 
 (test (begin (hash-table-union! ht-eqv2 ht-fixnum)
              (hash-table=? default-comparator ht-eqv2 ht-fixnum))
-      #t)
+      #f)
 
 (test (begin (hash-table-intersection! ht-eqv2 ht-fixnum)
              (hash-table=? default-comparator ht-eqv2 ht-fixnum))
-      #t)
+      #f)
```

Comment:
`hash-table=?` requires both hashtables to have the same equivalence function, but they are `eqv?` and `=` respectively.